lib/ebay_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module EbayClient
-  VERSION = '0.2.0.981'
+  VERSION = '0.2.0.982'
```

Comment:
the last number in the version reflects the ebay api version used - if so the version should be `0.2.1.981`